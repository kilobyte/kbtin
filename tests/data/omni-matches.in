#act {%1} {#show {act: [$1]}}
#act {%2} {#show {act: [$2]}}
#act {^%1$} {#show {act: [$1] be}}
#act {^%1} {#show {act: [$1] b}}
#act {%1$} {#show {act: [$1] e}}
#act {^$} {#show {act: empty}}
#sub {^$} {sub: empty}
#run a //selfpipe
foo

#del 1 #end
