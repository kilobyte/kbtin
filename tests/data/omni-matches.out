act: [foo] e
act: [foo]
act: [foo]
act: [foo] be
act: [foo] b
foo
act: [] e
act: []
act: []
act: empty
act: [] be
act: [] b
sub: empty

#SESSION 'a' DIED.
